version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "0.5.7"
+version in ThisBuild := "0.6.0"
```

Comment:
This file should be reverted.  We use a merge script which tags and updates the `version.sbt` file, so manual changes are not necessary.  This script currently doesn't work with private repos, but that's on our TODO list to fix.  Since I have added the `version: breaking` label to this PR, the version will be incremented to `1.0.0` upon merge.